simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java

```diff
@@ -301,6 +326,10 @@ private static Field findPropertyField(Class clazz, String property) {
         }
     }
 
+    private static void throwBindException(String msg, Object... args) {
```

Comment:
Removed the method https://github.com/hazelcast/hazelcast-simulator/commit/9f40ef6